docs/security/user-certs.md

```diff
@@ -25,36 +35,23 @@ Your **name** and **email** address are encoded into the signed certificate and
 You will have to obtain your user certificate using the [CILogon web UI](https://cilogon.org/).
 Follow the steps below to get an user certificate:
 
-1. First, either search for your institution and select it or scroll through list and do the same. <br>![Institution Selection](/img/cilogon_select_idp.png)
+1. First, either search for your institution and select it or scroll through list and do the same. If your institution is not on the list, please contact your institution's IT support to see if they can support CILogon. <br>![Institution Selection](/img/cilogon_select_idp.png).  
```

Comment:
There's some extra trailing space at the nd of the sentence. Could you also put the sentence on a newline?

Reply:
done